torch/quantization/quantize.py

```diff
@@ -220,6 +226,16 @@ def quantize_qat(model, run_fn, run_args, qconfig_dict=None):
     convert(model)
     return model
 
+def quantize_dynamic(model, run_fn=None, run_args=None, qconfig_dict=None):
+    r"""Converts a float model to dynamic quantized model. Do dynamic training and output a quantized model.
+    """
+    model.eval()
+    model = prepare_dynamic(model, qconfig_dict)
+    if run_fn is not None:
```

Comment:
I think we can remove `run_fn` and `run_args`?

Reply:
Right: we don't need the evaluation/calibration for the dynamic quantization path.

Reply:
Updated.